dask_jobqueue/tests/test_pbs.py

```diff
@@ -247,26 +236,19 @@ def test_adaptive_grouped(loop):
     ) as cluster:
         cluster.adapt(minimum=1)  # at least 1 worker
         with Client(cluster) as client:
-            start = time()
-            while not (cluster.pending_jobs or cluster.running_jobs):
-                sleep(0.100)
-                assert time() < start + QUEUE_WAIT
+            client.wait_for_workers(1)
 
             future = client.submit(lambda x: x + 1, 10)
             assert future.result(QUEUE_WAIT) == 11
 
             start = time()
-            while not cluster.running_jobs:
-                sleep(0.100)
-                assert time() < start + QUEUE_WAIT
-
-            start = time()
-            processes = cluster.worker_processes
+            processes = cluster.example_job.worker_processes
             while len(client.scheduler_info()["workers"]) != processes:
                 sleep(0.1)
                 assert time() < start + QUEUE_WAIT
 
 
+@pytest.mark.xfail(reason="adapt doesn't yet have cores/memory")
```

Comment:
This is a regression until we move this into core.  Are we ok with losing this functionality short term?

cc @guillaumeeb